src/library/Uno.Toolkit.Material/Styles/Controls/v2/NavigationBar.xaml

```diff
@@ -178,15 +178,85 @@
 
 	<!--#region Default Material NavigationBar-->
 	<Style x:Key="MaterialMainCommandStyle"
-		   BasedOn="{StaticResource MaterialAppBarButtonStyle}"
-		   TargetType="AppBarButton">
-		<Setter Property="Foreground" Value="{ThemeResource NavigationBarMainCommandForeground}" />
-		<!-- Native back icons are used on mobile platforms -->
-		<not_mobile:Setter Property="Icon" win:Value="{x:Null}">
-			<not_win:Setter.Value>
-				<PathIcon Data="{ThemeResource NavigationBarBackIconData}" />
-			</not_win:Setter.Value>
-		</not_mobile:Setter>
+       TargetType="AppBarButton">
+		<Setter Property="Background" Value="{ThemeResource AppBarButtonBackground}" />
+		<Setter Property="Foreground" Value="{ThemeResource AppBarButtonForeground}" />
+		<Setter Property="BorderBrush" Value="{ThemeResource AppBarButtonBorderBrush}" />
+		<Setter Property="HorizontalAlignment" Value="Left" />
+		<Setter Property="VerticalAlignment" Value="Top" />
+		<Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />
+		<Setter Property="FontWeight" Value="Normal" />
+		<Setter Property="Width" Value="68" />
+		<Setter Property="UseSystemFocusVisuals" Value="{StaticResource UseSystemFocusVisuals}" />
+		<Setter Property="FocusVisualMargin" Value="-3" />
+		<Setter Property="AllowFocusOnInteraction" Value="False" />
+		<Setter Property="Content" Value="{ThemeResource NavigationBarBackIconData}" />
```

Comment:
I'm noticing we don't mention the NavigationBarBackIconData value in the list of Lightweight Styling resources here: https://github.com/unoplatform/uno.toolkit.ui/blob/main/doc/controls/NavigationBar.md#lightweight-styling

We should add it to the table